docs/libcurl/curl_easy_ssls_export.md

```diff
@@ -150,6 +150,12 @@ int main(void)
   if(curl) {
     curl_easy_setopt(curl, CURLOPT_SHARE, share);
 
+    /* run a transfer, all TLS sessions received will be added
+     * to the share. */
+    curl_easy_setopt(curl, CURLOPT_URL, "https://www.curl.se/");
```

Comment:
```suggestion
    curl_easy_setopt(curl, CURLOPT_URL, "https://curl.se/");
```
use the canonical URL? (I only used `www.` to test LibreSSL in the Windows build.)

Reply:
okok.